README.rst

```diff
@@ -0,0 +1,139 @@
+|Version| |Layers| |Build Status| |Code Climate| |Coverage Status|
+
+Plankton
+========
+
+Plankton is a Python framework for simulating hardware devices. It is
+compatible with both Python 2 and 3.
+
+Plankton can be run directly using Python 2.7 or >= 3.4, or using a
+prepackaged Docker image that includes all dependencies. See relevant
+usage sections for details.
+
+Resources: -
+`GitHub <https://github.com/DMSC-Instrument-Data/plankton>`__ -
+`DockerHub <https://hub.docker.com/r/dmscid/plankton/>`__ -
+`Dockerfile <https://github.com/DMSC-Instrument-Data/plankton/blob/master/Dockerfile>`__
```

Comment:
This looks broken now... used to be a bullet point list. I actually like it better side-by-side, but then the first `-` should be removed. And potentially use a `--` or `|` as the separator between links?